packages/example-getting-started/docs/7-controller.md

```diff
@@ -0,0 +1,71 @@
+### Create your controller
```

Comment:
OOPS accidentally added it when rebase, deleted it in https://github.com/strongloop/loopback-next/pull/1067/commits/fc6b3d15769a592388540f2e23f60bba339e4d4d